testdata/build-test-registry.sh

```diff
@@ -7,23 +7,23 @@ set -o pipefail
 help="
 image-registry.sh is a script to stand up an image registry within a cluster.
```

Comment:
Has the script name changed and this line needs to be updated?

Reply:
Fixed, thanks!